packages/shared/src/hooks/auth/useOnboarding.tsx

```diff
@@ -33,7 +33,7 @@ export const useOnboarding = (): UseOnboarding => {
 
   return {
     shouldShowAuthBanner,
-    isOnboardingReady: isActionsFetched && isAuthReady,
+    isOnboardingReady: isAuthReady && (isActionsFetched || !user),
```

Comment:
What's the issue here I missed it maybe?

Reply:
There is an edge case where the loading state is there endlessly. One the condition is whether the returned value here is true. The loading never stop since anonymous user would not have actions hence not yet fetched.

Reply:
Because of that, the event listeners are not rendered which is needed to process the callback page messages on login/sign up.